tests/test_sphinx_autodoc_typehints.py

```diff
@@ -1,10 +1,13 @@
+import pathlib
 import pytest
 import sys
+import textwrap
 from typing import (
     Any, AnyStr, Callable, Dict, Generic, Mapping, Optional, Pattern, Tuple, TypeVar, Union)
 
 from sphinx_autodoc_typehints import format_annotation, process_docstring
 
+
```

Comment:
Just FYI, the try...except import block is the fourth group of imports – no extra empty lines!